cogs/games.py

```diff
@@ -23,15 +23,225 @@
 
 # Other utilities
 import re
+import os
+import sqlite3
+from time import time
+import pickle
+import random
+import asyncio
+from typing import Optional
+from functools import partial
 from .utils.dice_roll import dice_roll
 from .utils.clamp_default import clamp_default
+from .utils.hangman import HangmanState
+from .currency import HANGMAN_REWARD
 
 ROLL_PATTERN = re.compile(r'^(\d*)d(\d*)([+-]?\d*)$')
+CATEGORY_SYNONYMS = {
+    "movies": "movie",
+    "kino": "movie",
+    "elements": "element",
+    "countries": "country",
+    "animals": "animal",
+}
 
 
 class Games(commands.Cog):
-    def __init__(self, bot):
+    def __init__(self, bot, hangman_tbl_name: str):
         self.bot = bot
+        self.hm_cool_win: int = bot.config.games["hm_cool_win"]
+        self.hm_norm_win: int = bot.config.games["hm_norm_win"]
+        self.hm_timeout: int = bot.config.games["hm_timeout"]
+        self.hm_locks: dict[discord.TextChannel, asyncio.Lock] = dict()
+        with open(f"{os.getcwd()}/data/premade/{hangman_tbl_name}.obj",
+                  "rb") as hangman_pkl:
+            self.hangman_dict: dict[str, tuple[list[tuple[str, Optional[str]]],
+                                               str]] = pickle.load(hangman_pkl)
+
+    def help_str(self):
+        cat_list: str = ", ".join(
+            f"`{hm_cat}` (length: {len(self.hangman_dict[hm_cat][0])})"
+            for hm_cat in sorted(self.hangman_dict.keys()))
+        return ("rules: 5 wrong guesses are allowed, guesses "
+                "must be either the entire correct word or a "
+                "single letter (interpreted in a case insensitive manner)\n"
+                f"here is a list of valid category commands: {cat_list}\n"
+                "other commands are: `quit` (stops current game) "
```

Comment:
I'd write this like you did in the docsting (like `Quit an ongoing game by typing "?{hm|hangman} quit".`)

Reply:
resolved in `d6bf98a`.